packages/reaction-core/server/methods/shop.js

```diff
@@ -424,12 +453,8 @@ Meteor.methods({
       });
     }
     // create newTags
-    newTag.isTopLevel = !currentTagId;
-    newTag.shopId = ReactionCore.getShopId();
```

Comment:
Hello, because this happens automatically [here](https://github.com/reactioncommerce/reaction/blob/development/packages/reaction-schemas/common/schemas/tags.js#L32)